_data/sections/contribution-limits/legends.yml

```diff
@@ -36,5 +36,8 @@
 - name: Party2Party
   type: singular
   scale:
-    - label: "Forbidden but funds get transferred between the two entities following internal procedures. And this is a sentence to act as a space filler."
+    - label: "This is more nuanced than donating.  Funds are able to be transferred
+      from state to local parties via internal processes.
+
+      And this is a sentence to act as a space filler."
```

Comment:
Is this sentence left in here intentionally? "And this is a sentence to act as a space filler"